src/build_workflow/build_target.py

```diff
@@ -79,6 +79,12 @@ def compatible_component_versions(self) -> List[str]:
             + list(map(lambda version: BuildTarget.__qualify_version(version + ".0", self.qualifier, True), self.patches))
         )
 
+    @property
```

Comment:
Last one, I promise, should `compatible_component_versions` now we renamed to `compatible_opensearch_component_versions`?

Reply:
After discussing, we decided that OpenSearch Dashboards plugins will need to manually bump their version to add the qualifier and we won't allow for `x.y.z.n` plugins to build with OpenSearch Dashboards `x.y.z-qualifier`.

That said I will remove the `opensearch_dashboards_component_versions` and then we can probably just keep `compatible_component_versions` the same name.

Reply:
Updated